packages/components/psammead-heading-index/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@bbc/psammead-heading-index",
+  "version": "1.0.0",
```

Comment:
Should this be an alpha version since this is an alpha component?

Reply:
Of course!